d3a_api_client/redis_aggregator.py

```diff
@@ -28,7 +28,7 @@ class RedisAPIException(Exception):
 class RedisAggregator:
 
     def __init__(self, aggregator_name, accept_all_devices=True,
-                 redis_url='redis://localhost:6379'):
+                 redis_url="redis://localhost:6379"):
```

Comment:
Outside of the scope of the PR, but since it won't take long, could you please take the time and refactor this to a global constant named LOCAL_REDIS_URL or similar in order to be reusable ? Thanks!